CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ❤️ **Donate:** Enjoying MagicMirror²? [Please consider a donation!](https://magicmirror.builders/donate) With your help we can continue to improve the MagicMirror²
 
+## [20.11.1] - 2020-06-20
```

Comment:
I guess you will have let @MichMich to decide if a bugfix release will happen for that...

Reply:
Ok. I only added this because the guidelines suggested me to edit the changelog. I'm happy to revert this change if needed.